portfolio/src/main/webapp/script.js

```diff
@@ -29,6 +29,30 @@ function addRandomShow() {
 
 async function getName() {
   const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('hello-container').innerText = quote;
+  const name = await response.text();
+  document.getElementById('hello-container').innerText = name;
+}
+
+async function getCommentStats() {
```

Comment:
Should this just be `getComments` or should this function be showing statistics of some sort as well?